### PR TITLE
Add Exa AI Project Tutorial

### PR DESCRIPTION
**Pull request by exrlla**
Status: ⚠️ 
- [x] What is tokenization - Feedback from Sarah
- [x] Adding the TikTok visual - @ksolis3197 
- [ ] Reviews

### CODE REVIEW

File: projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+Have you ever read something amazing, then completely blanked out on where you read it or who wrote it?
+
+Imagine there was a search engine that could re-discover exactly what you're thinking about. Now, it's possible with machine learning. More specifically **Natural Language Processing (NLP)**, a branch of machine learning that teaches computers to understand, interpret, and generate human language. 
+In this tutorial, we'll build a custom search engine using an API with LLM capabilities! 🚀 
+
+ **LLMs** or Large Language Models are a powerful tool within the broader field of NLP. These models are trained on insane amounts of data and can grasp the intricacies of human language. 🤯 A popular LLM that you might already be using is GPT-3. 
+
+<RoundedImage link="https://raw.githubusercontent.com/codedex-io/projects/main/projects/build-a-custom-search-engine-with-exaai/token.gif" description="Tokenization Example" />
+
+LLMs leverage deep learning techniques to generate text, translate languages, answer questions, and so much more. The model works by estimating the probability of a token or a sequence of tokens in a sequence.
```
Comment (ksolis3197) [inline projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx:37]: (a token refers to a unit of language extracted from a larger piece of text)

File: projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+```py
+response = exa.search( "best pizza in Brooklyn",
+    num_results=10,
+    start_published_date="2023-05-01", 
+    category="tweet", 
+    use_autoprompt= True, )
+```
+
+This is where the fun customization begins! ✨💖 For this tutorial, you'll learn how to search TikTok to obtain top coffee drink accounts! 
+
```
Comment (ksolis3197) [inline projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx:138]: insert visual here

File: projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+<BannerImage link="https://raw.githubusercontent.com/codedex-io/projects/main/projects/build-a-custom-search-engine-with-exaai/header.gif" description="Title Image" uid={true}/>
+
+**Prerequisites:** Python, Command Line
+**Versions:** Python 3.10
+**Read Time:** 45 minutes
+
+## Introduction
+
+Have you ever read something amazing, then completely blanked out on where you read it or who wrote it?
```
Comment (Dusch4593) [inline projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx:29]: I thought Sarah said this sentence didn't make sense?

**Review by Dusch4593 — COMMENTED** [review]
@exrlla @ksolis3197 I just made some changes in https://github.com/codedex-io/projects/pull/132/commits/46795697ff3076be5347d2d8ee3cf7d2e6879d06, https://github.com/codedex-io/projects/pull/132/commits/64131632d6dc626b469700a5365e326e92ac81f6, https://github.com/codedex-io/projects/pull/132/commits/b7107ca83e1a97a24ef0de91bec824fa068d02fe, and https://github.com/codedex-io/projects/pull/132/commits/38da8dc4e7e39f7160f941409cfc76e7e806da56.

I also left one comment. Once it is resolved, I think this is good to merge!

cc: @asiqurrahman

File: projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx (hunk 3 above)
Comment (exrlla) [reply projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx:29]: @ksolis3197 want to remove it? we can change this hook to be something else

Comment (ksolis3197) [reply projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx:29]: I rewrote it. Does it still not make sense?

Comment (Dusch4593) [reply projects/build-a-custom-search-engine-with-exaai/build-a-custom-search-engine-with-exaai.mdx:29]: Ok I realize the change you made, @ksolis3197.

**Review by Dusch4593 — APPROVED** [review]
LGTM!

**Pull request merged**